sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/meta/provider/test/MockedBoundedTable.java

```diff
@@ -35,6 +33,7 @@
 import org.apache.beam.sdk.values.Row;
 
 /** Mocked table for bounded data sources. */
+@Experimental
 public class MockedBoundedTable extends MockedTable {
```

Comment:
Same comment about mocks here, this should be a `TestBoundedTable`

Reply:
Done